modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java

```diff
@@ -151,6 +160,25 @@ private String _getURLWithBackURL(String url) {
 		return _http.setParameter(url, "p_l_back_url", _currentURL);
 	}
 
+	private DropdownItem _toDropdownItem(
+		ContentDashboardItemAction contentDashboardItemAction) {
+
+		DropdownItem dropdownItem = new DropdownItem();
+
+		dropdownItem.setData(
+			HashMapBuilder.<String, Object>put(
+				"action", "showMetrics"
+			).put(
+				"fetchURL", contentDashboardItemAction.getUrl()
```

Comment:
@carloslancha I finally chose `fetchURL` since It's already in master

Reply:
![image](https://media.giphy.com/media/yoJC2GnSClbPOkV0eA/giphy.gif)

Reply:
All this action could no be fetchURL, some of them will be EditURLs or ViewURLs, we should add a type to check this specific show in InfoPanel URLs

Reply:
Also, maybe the order is important? Could we spec a specific order for these actions?

Reply:
>All this action could no be fetchURL, some of them will be EditURLs or ViewURLs

This is an specific `data-attribute` that will be received in `ActionsComponentPropsTransformer` together with `action`, and we're using it in `SidebarPanel` to fetch the data/content.

If you want to make the item a link to certain url it should be set as href, not as a `data-attribute`.